torchtune/modules/loss/ce_chunked_output_loss.py

```diff
@@ -12,16 +12,16 @@
 
 class CEWithChunkedOutputLoss(torch.nn.Module):
     """
-    CE with chunked outputs that saves memory by only upcasting one chunk at a time.
+    Cross-entropy with chunked outputs that saves memory by only upcasting one chunk at a time.
 
-    Since the model is trained with bf16, before running CE, we have to upcast
+    Whenever the model is trained with bf16, before running CE, we have to upcast
     it to fp32 for better accuracy and stability. When upcasting happens, the memory usage doubles.
     Models like llama3 have large vocabulary size and, therefore, have a large output
-    result (bsz, num_tokens, vocab_size). If we chunk on the token level, you can still compute
+    tensor of shape ``(bsz, num_tokens, vocab_size)``. If we chunk on the token level, you can still compute
     the cross entropy normally, but upcasting only one chunk at a time saves considerable memory.
 
     The CE and upcasting have to be compiled together for better performance.
-    When using this class, we recommend using torch.compile only on the method `compute_cross_entropy`.
+    When using this class, we recommend using :func:`torch.compile` only on the method ``compute_cross_entropy``.
```

Comment:
Can you self reference method here?

Reply:
Was debating it but honestly it's like 5 lines below and not well-documented anyways so I thought it was overkill